lib/util.js

```diff
@@ -0,0 +1,70 @@
+const fs = require('fs');
+const exec = require('child_process').exec;
+const Docker = require('dockerode');
+const docker = new Docker();
+
+const MINUTES_IN_HOUR = 60;
+
+function nameMeetsCriteria(containerName, criteriaName) {
+    if (criteriaName === '*') {
+        return true
+    } else {
+        return containerName.includes(criteriaName)
+    }
```

Comment:
This is fine for PoC state, but better to use something like `minimatch` package to support real patters. 

Here you can play with minimatch package and this example to see how it works: https://npm.runkit.com/minimatch
```js
var minimatch = require("minimatch")

const paragraph = 'The quick brown fox jumps over the lazy dog.';
minimatch(paragraph, 'the quick*', {nocase: true})
```

Reply:
Done